connector/protobuf/src/test/resources/protobuf/functions_suite.proto

```diff
@@ -324,3 +325,19 @@ message Proto3AllTypes {
   }
   map<string, string> map = 13;
 }
+
+message WellKnownWrapperTypes {
+  google.protobuf.BoolValue bool_val = 1;
+  google.protobuf.Int32Value int32_val = 2;
```

Comment:
Could you add test where `int32_val` is not set. What should the Spark struct contain:
  * When `emit.default.values` is false (default)
  * When `emit.default.values` is true 
 Please comment on the expected behavior. 

Another similar test with `int32_val.value` set to `0` (default value).

Reply:
yeah, great suggestion. I added that test!

basically the internal `value` field should operate the same as any primitive, and when unwrapping it should unwrap that value correctly. I added a test that confirms every configuration of emit.default.values and unwrap.primitive.wrapper.types